tools/test-case-generators/format-request-map.json

```diff
@@ -513,7 +513,14 @@
       }
     },
     "no-image-info": true,
-    "overrides": {}
+    "overrides": {
+      "rhel-85": {
+        "blueprint": {}
+      },
+      "rhel-90-beta": {
+        "blueprint": {}
+      }
+    }
```

Comment:
Is this really needed for 8.5 and 9.0 beta? IIRC, the support for user customizations was backported to 8.5 and 9.o beta via https://github.com/osbuild/osbuild-composer/pull/2516

Reply:
The problem was the fdo settings are not supported for those versions. I got the next error:
`panic: boot ISO image type "edge-simplified-installer" contains unsupported blueprint customizations: 'FDO' is not allowed`